database/migrations/2018_06_19_025133_create_verify_users_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateVerifyUsersTable extends Migration
```

Comment:
It'll be good if we rename the file to `create_user_verification_tokens_table.php` and the class to `CreateUserVerificationTokensTable`.

Reply:
it's duplicate i did remove it .